docs/DeveloperGuide.md

```diff
@@ -511,34 +511,62 @@ testers are expected to do more *exploratory* testing.
 
 ### Deleting a client
 
-1. Deleting a client in the Client List View
+1. Deleting a client in the client list view
 
    1. Prerequisites: List all clients using the `list` command. Multiple clients in the list.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. 
 
    1. Test case: `delete 0`<br>
       Expected: No client is deleted. Error details shown in the status message. Status bar remains the same.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-2. Deleting a client in the Transaction List View
+2. Deleting a client in the transaction list view
    1. Prerequisites: List all transactions of a client, such as the first, using the `listt 1` command.
   
     1. Test case: `delete 0`<br>
-      Expected: No client is deleted. Error detail informing environemnt discreprancy shown in the status message.
+      Expected: No client is deleted. Error detail informing of environment discrepancy shown in the status message.
 
     2. Test case: `delete 1`<br>
-      Expected: No client is deleted. Error detail informing environemnt discreprancy shown in the status message.
+      Expected: No client is deleted. Error detail informing of environment discrepancy shown in the status message.
 
     3. Test case: `delete x`, where `x` is exactly 1 more than transaction list size<br>
       Expected: No client is deleted. Error detail informing out of range index shown in the status message.
```

Comment:
i think this is also the error from the environment discrepency, because we check for the environment before checking the index.

Reply:
Then maybe no need to include this test case? @trpmy shall we remove this test case

Reply:
We can relocate this test case to be testing delete under client view then? `delete` and `delete x` and `delete 0` all yields different error messages, so it'd be nice to have them under that section